restAPI/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
```

Comment:
We may want to change the location of this file. You will get a better idea once we discuss the code structure we try to abide by here

Reply:
Sure, I think this is not yet fixed in latest PR as well, so we can discuss about it in next review